run_models.py

```diff
@@ -0,0 +1,247 @@
+          input                                   input                                            output                                                  output
+==========================================================================
+           ---                                         ---                         Formulates path to OFLS directory       Formulated path to OFLS directory 
+Path to OFLS directory                    ---                                   Path to OFLS directory                         Path to OFLS directory
+           ---                           Path to output directory     Formulates path to OFLS directory               Path to output directory
+Path to OFLS directory      Path to output directory              Path to OFLS directory                         Path to output directory
+
+   The formulated path to the OFLS directory is:  /data/acis/LoadReviews/<year>/<load>/ofls<version letter>
+   It is based upon the load week and version letter which are  required input.
+
+ Example command line for the 1DPAMZT model:
+
+    /proj/sot/ska3/flight/bin/dpa_check APR2825 --oflsdir  /data/acis/LoadReviews/2025/APR2825/ofls -nlet_file nlet_file "b"
+
+ Initial: August 23, 2024
+           Gregg Germain
+
+"""
+
+# Set up the parser
+rmparser = argparse.ArgumentParser()
+rmparser.add_argument("load_name", help="Name of the Review load including letter (e.g. AUG0924A)")
+rmparser.add_argument("--oflsdir", help="Full path to the OFLS directory of the review load")
+rmparser.add_argument("--out", help="Full path output  directory for the out_* files (e.g. /data/acis/LoadReviews/2025/APR2825/ofls)")
+rmparser.add_argument("-b", help="If in the command line, specifies if the load is an interrupt load",  action="store_true")
+
+rmparser.add_argument("--nlet_file", help="Full Path to the Non Load Event Tracking file to be used", default="/data/acis/LoadReviews/NonLoadTrackedEvents.txt" )
+rmparser.add_argument("--verbose", help="Indicates verbosity of debug comments", default = 0)
+
+rmparser.add_argument("-t", help="Flag to allow/prevents writing the out_<model> directories to the web page", action="store_true")
+
+# Get the arguments
+args = rmparser.parse_args()
+
+# Capture the name of the load.  e.g. DEC0224A
+load_name = args.load_name
+
+# Extract the year and load version letter from the load name
+year = "20"+load_name[5:-1]
+
+# load is the weekly load name without the letter version
+load = load_name[:-1]
+
+# ver is the version letter of the load (e.g. the "A" of AUG1224A)
+ver = load_name[-1]
+
+# OFLSDIR - If the user specified an OFLS directory path then use that one.
+# Otherwise form the usual production path using the year, the load and the version
+if args.oflsdir:
+    # Use the supplied directory
+    oflsdir = args.oflsdir
+else:
+    # Formulate the PRODUCTION directory
+    oflsdir = "/".join(("/data/acis/LoadReviews/" + year, load, "ofls"+ver.lower()))
```

Comment:
You have already imported `os` so I would tend to prefer `os.path.join` for this job